cmd/otelcol/Dockerfile

```diff
@@ -10,6 +10,7 @@ COPY migratecheckpoint /
 RUN chmod 755 /otelcol
 RUN chmod 755 /translatesfx
 RUN chmod 755 /migratecheckpoint
+RUN echo "splunk-otel:x:999:999::/:" > /etc_passwd
```

Comment:
Just a nit, but the deb/rpm creates and runs the collector service as the `splunk-otel-collector` user/group.  Should we use the same name for consistency?